facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -320,8 +322,11 @@
     </table>
 }
 
-@layout.CollectionEmail.fromPressedPage(page).collections.zipWithIndex.map {
-    case (collection: EmailContentContainer, index) if FlagshipEmailContainer.isFlagshipContainer(collection.containerId) => { @renderFlagshipContainer(collection) }
+@layout.CollectionEmail.fromPressedPage(page).collections.filterNot(collection => collection match {
```

Comment:
@tomrf1 I moved the logic here... I guess it's better?

Reply:
yep, at least it's all in once place, thanks!
Also, I think it can be shortened to:
```
collections.filterNot {
  case c: EmailContentContainer => FlagshipEmailContainer.isFlagshipContainer(c.containerId) && !FlagshipEmailContainer.displayFlagshipContainer()
  case _ => false
}
```

Reply:
yeah that's what IntelliJ was telling me - but then I get `Not applicable to (A => Boolean)`! might be a weird Twirl edge case?